cernopendata/modules/theme/assets/semantic-ui/js/records/components/RequestRecord.js

```diff
@@ -13,8 +13,11 @@ const RequestRecordApp = ({ recordId, availability, files, size }) => {
     if (!confirmed) return;
     setLoading(true);
     try {
-      await axios.post(`/record/${recordId}/stage`, { email: email.trim() });
-      window.location.reload();
+        await axios.post(`/record/${recordId}/stage`, {
```

Comment:
do we have some throttling or debounce on this? To avoid multiple calls on the endpoint?